netbox/ipam/tables/l2vpn.py

```diff
@@ -0,0 +1,38 @@
+import django_tables2 as tables
+
+from ipam.models import *
+from ipam.models.l2vpn import L2VPN, L2VPNTermination
+from netbox.tables import NetBoxTable, columns
+
+__all__ = (
+    'L2VPNTable',
+    'L2VPNTerminationTable',
+)
+
+
+class L2VPNTable(NetBoxTable):
```

Comment:
Should probably add columns for the import and export route targets too. Should be able to replicate what we did for `VRFTable`.